Server/intake.php

```diff
@@ -170,32 +170,48 @@
     // Loot
     if (!empty($_POST['loot']) && $hunt_id > 0) {
         $loot_array = [];
-        $gold_array = [15, 47, 106, 138, 191, 194, 210, 226, 227, 260, 261, 262, 264, 265];
         foreach ($_POST['loot'] as $loot_item) {
-            $loot_item['amount'] = str_replace(",", "", $loot_item['amount']);
+
+            // Amount checks
             if (!is_numeric($loot_item['amount']) || $loot_item['amount'] < 1) {
                 continue;
             }
 
+            // Lucky checks
             $lucky = null;
-            if (!empty($loot_item['lucky'])) {
-                $lucky = $loot_item['lucky'] === 'true' ? 1 : 0;
-            }
+            if (!empty($loot_item['lucky'])) { $lucky = $loot_item['lucky'] === 'true' ? 1 : 0; }
 
-            $query = $pdo->prepare("SELECT id FROM loot WHERE name LIKE ?");
-            $query->execute(array($loot_item['name']));
-            $loot_id = $query->fetchColumn();
+            // Hitgrab item id checks
+            $hg_item_id = null;
+            if (is_numeric($loot_item['id'])) { $hg_item_id = $loot_item['id']; }
 
-            if (in_array($loot_id, $gold_array)) { // Blocking gold, there could be multiples of it, and not accurate
-                continue;
-            }
+            // Plural name checks
+            $plural_name = null;
+            if (!empty($loot_item['plural_name'])) { $plural_name = $loot_item['plural_name']; }
 
+            // Single name checks
+            $single_name = null;
+            if (!empty($loot_item['name'])) { $single_name = $loot_item['name']; }
+
+            // Get mhct id
+            $query = $pdo->prepare("SELECT MIN(id) FROM loot WHERE name LIKE ? OR name LIKE ? OR hg_item_id = ?");
```

Comment:
A little worried about this finding lots of items because of so many nulls for now. 

`SELECT MIN(id) FROM loot WHERE name LIKE coalesce(?, 'not found') OR name LIKE coalesce(?, 'not found') OR hg_item_id = ?`

It might not be a thing depending on how null values work with like. Or we change the likes to equals.

Reply:
wasn't a thing when i tested it.